pubspec.yaml

```diff
@@ -1,12 +1,12 @@
 name: stream_channel
-version: 1.6.8
```

Comment:
given that there are not any breaking changes yet I would just do a dot release

also, please add a corresponding changelog entry

Reply:
Done